woocommerce/woocommerce
# Fix links popover disappearing rapidly after appearing

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




Fix links popover disappearing rapidly after appearing

This reverts the fixes implemented in #45608, since the original issue that is fixes is not reproducible anymore in latest WordPress 6.5.2.

The hook that is being removed in this PR was causing the issue #45663.

Closes #45663 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Products > Add
3. If you have the gutenberg plugin activated, please disable it.
4. In the summary field, add a link, and afterwards click it to edit it. The popover should appear normally and you should be able to edit the link
5. Do the same for the Description field, as it's a slightly different implementation.
6. Install the gutenberg plugin version 17.9, open the product editor, and check that you see an alert message warning that there are incompatibilities with this version.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
